src/tokenize/tokenize.ts

```diff
@@ -19,101 +17,35 @@ import {
   XOM_REGEX_NUMBER,
 } from "./constants";
 import {
-  CountAndHoursRow,
-  CourseRow,
-  HDocument,
-  HRow,
+  type CountAndHoursRow,
+  type CourseRow,
+  type HRow,
   HRowType,
-  HSection,
+  type HSection,
   HSectionType,
-  MultiCourseRow,
-  RangeBoundedRow,
-  RangeLowerBoundedRow,
-  RangeUnboundedRow,
-  TextRow,
-  TokenizedCatalogEntry,
-  WithExceptions,
+  type MultiCourseRow,
+  type RangeBoundedRow,
+  type RangeLowerBoundedRow,
+  type RangeUnboundedRow,
+  type TextRow,
+  type TokenizedCatalogEntry,
+  type WithExceptions,
```

Comment:
Yep, changed as recommended. It will be a bit annoying having to edit that when adding new types (couldn't find a way to get `prettier` to do it automatically though)